indexer/categories_holder.hpp

```diff
@@ -96,21 +96,22 @@ class CategoriesHolder
       {"it", 24},
       {"ja", 25},
       {"ko", 26},
-      {"mr", 27},
-      {"nb", 28},
-      {"nl", 29},
-      {"pl", 30},
-      {"pt", 31},
-      {"pt-BR", 32},
-      {"ro", 33},
-      {"ru", 34},
-      {"sk", 35},
-      {"sv", 36},
-      {"sw", 37},
-      {"th", 38},
-      {"tr", 39},
-      {"uk", 40},
-      {"vi", 41},
+      {"lv", 27},
```

Comment:
does it have to be in alphabetical order? (looks like no as `en` goes first)

does the ids change break compatibility with older map files?

Reply:
No specific order requirements. Alphabetic sorting is used for readers' comfort.

Reply:
@vng why do we have hardcoded ids here if they don't matter? is it possible to change it to just provide a list of supported langs without ids?

Reply:
Good question. Probably, the initial idea was to make category lang codes the same as StringUtf8Multilang lang codes ..